### ISSUE TITLE
not found in npm registry

### ISSUE THREAD

**Issue by vasco3** [ArtskydJ/spell-check-cli#1]
https://registry.npmjs.org/spellcheck-cli

```

npm install spellcheck-cli --global
npm ERR! Darwin 15.4.0
npm ERR! argv "/usr/local/Cellar/node/5.0.0/bin/node" "/usr/local/bin/npm" "install" "spellcheck-cli" "--global"
npm ERR! node v5.0.0
npm ERR! npm  v3.3.9
npm ERR! code E404

npm ERR! 404 Registry returned 404 for GET on https://registry.npmjs.org/spellcheck-cli
npm ERR! 404
npm ERR! 404 'spellcheck-cli' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/br-mb004/npm-debug.log
```

**Comment by ArtskydJ** [comment]
It's incorrectly documented, it should be `spell-check-cli` (extra hyphen).

Thanks!

**Comment by ArtskydJ** [comment]
Published as 1.0.2.
